interface-definitions/vpn_ipsec.xml.in

```diff
@@ -768,6 +768,16 @@
                     <children>
                       #include <include/ipsec/authentication-id.xml.i>
                       #include <include/ipsec/authentication-x509.xml.i>
+                      <leafNode name="eap-id">
```

Comment:
Can you please add the completionHelper node for value any?

While doing this please add an explicit valueHelp for any.

Are all EAP IDs supported? Or only alphanumeric with @? We should think about a proper constraint, too.

Reply:
Thanks, I have added the helper nodes requested.

Not too sure for constraints...
For my usage with EAP-TLS the identity is a SAN of the client certificate. Either a FQDN or RFC822 name, supporting wildcards `(IE: *@vyos.net)`.

Reply:
```xml
<constraint>
  <regex>[[:ascii:]]</regex>
</constraint>
```

should do the trick

Reply:
Thank you, I have added the constraint.